cmd/expiration-mailer/main_test.go

```diff
@@ -523,7 +515,7 @@ func TestCertIsRenewed(t *testing.T) {
 			DER:            certDer,
 		}
 		fqdnStatus := &core.FQDNSet{
-			SetHash: testData.FQDNHash,
+			SetHash: sa.HashNames([]string{testData.DNS}),
```

Comment:
I would like to see some of these test cases use more than one DNS name or at least be covered by tests for `sa.HashNames`.